clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/ops/CloudFoundryOperationUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 Pivotal, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.cloudfoundry.deploy.ops;
+
+import com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.v3.ProcessStats;
+
+class CloudFoundryOperationUtils {
+  static String describeProcessState(ProcessStats.State state) {
+    switch(state) {
+      case STARTING:
+        return "is still starting";
+      case CRASHED:
+        return "crashed";
```

Comment:
maybe "has crashed" for consistency?

Reply:
The text of a message coming from this reads: "Failed to start 'demo' which instead crashed".